packages/common-library/src/common_library/network.py

```diff
@@ -7,3 +8,30 @@ def is_ip_address(host: str) -> bool:
         return True
     except ValueError:
         return False
+
+
+def redact_url(url: str) -> str:
```

Comment:
Q: Is `redact` a usual term in CS for this? 
This function is used to log urls w/o sensitive information, right? why not state that in the naming of the function?  I cannot deduce that from "redact"